packages/story-editor/src/components/panels/shared/hotlinkModal.js

```diff
@@ -40,12 +50,39 @@ import {
 import { useAPI } from '../../../app/api';
 import useCORSProxy from '../../../utils/useCORSProxy';
 
+const DOCS_URL =
+  'https://wp.stories.google/docs/troubleshooting/common-issues/';
+
 const InputWrapper = styled.form`
   margin: 16px 4px;
   width: 470px;
   height: 100px;
 `;
 
+function CORSMessage() {
```

Comment:
Does this make sense to be in it's own file?

Reply:
I don't see a reason to move this to a separate file. It's specific to this file and not used elsewhere. Plus it's super simple and basically a simple string.